vmdb/app/models/ems_microsoft.rb

```diff
@@ -1,6 +1,7 @@
-
 $:.push(File.expand_path(File.join(Rails.root, %w{.. lib Scvmm})))
 
+require 'gssapi'
```

Comment:
Ok, I delayed the require of gssapi until that method. Does that address your concern? Otherwise, I'm not sure what you mean.